custom/src/main/java/com/splunk/opentelemetry/SplunkConfiguration.java

```diff
@@ -71,6 +64,60 @@ public Map<String, String> defaultProperties() {
   public Config customize(Config config) {
     ConfigBuilder builder = config.toBuilder();
 
+    String metricsImplementation = config.getString(METRICS_IMPLEMENTATION);
+    if (metricsImplementation == null) {
+      metricsImplementation = "micrometer";
+      builder.addProperty(METRICS_IMPLEMENTATION, "micrometer");
+    }
+
+    if ("micrometer".equals(metricsImplementation)) {
+      // TODO: warn that micrometer metrics are deprecated
+      // log.info(
+      //     "Micrometer metrics are deprecated, OpenTelemetry metrics will become the default
+      // implementation in the next release.");
+
+      // by default no otel metrics are exported
+      addIfAbsent(builder, config, "otel.metrics.exporter", "none");
```

Comment:
This will enable otel metrics if `splunk.metrics.implementation` is set to `opentelemetry`, even if `splunk.metrics.enabled` is false (same for the list of instrumentations below).

Perhaps we should keep them all disabled in `defaultProperties()` and only enable them when a particular implementation is chosen and metrics are enabled?